doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -45,6 +43,20 @@ Field         | Type       | Meaning
 `records`     | [`bigint`] | The number of records in the arrangement.
 `batches`     | [`bigint`] | The number of batches in the arrangement.
 
+### `mz_cluster_replica_metrics`
```

Comment:
This looks fine but also I don't really know anything about it!

Reply:
Oh, I just shifted this one around to be close to the other cluster-related table, rather than dangling all alone at the bottom of the list!

Reply:
Ah, that's me not paying attention, sorry!